### DISCUSSION TITLE
Open Wikipedia page in default web browser via right-click from inside GoldenDict?

### DISCUSSION THREAD

**Discussion by michaelbeijer** [xiaoyifang/goldendict-ng#451] [Ideas]
It would be cool if you could open a Wikipedia page in your default web browser via right-click in GoldenDict. Often, I want to click though to the page online so I can look at more stuff (language links mainly), and a quick way to do this would be cool.

Michael

**Comment by xiaoyifang** [comment]
this feature is supported .  right click on the **external** links and there would be an `open in external browser`

**Reply by michaelbeijer** [reply]
Am I right in assuming that this only works if the resource is in under the **Websites** tab (i.e., configured with "%GDWORD%"), but not if it is in the **Wikipedia** tab?

![wikipedia-searching-in-goldendict](https://user-images.githubusercontent.com/16622240/231303493-33aa4bc7-5e99-4dab-aec5-d945939878a3.png)

**Reply by xiaoyifang** [reply]
works in both tabs

**Reply by michaelbeijer** [reply]
Hmm, it doesn't seem to work in both in my copy of GoldenDict. I am currently running:

Goldendict 23.04.03-alpha.230409.c472b2cb
windows winnt 10.0.22623 Qt 6.4.3 x86_64-little_endian-llp64
Visual C++ Compiler: 193431943
Flags: MAKE_ZIM_SUPPORT MAKE_EXTRA_TIFF_HANDLER MAKE_CHINESE_CONVERSION_SUPPORT

**Reply by xiaoyifang** [reply]
which entry item in the wiki？

**Reply by xiaoyifang** [reply]
I think I got your idea.   

In the wiki article's body, only absolute url can be opened in external browser,  the relative url can not .

The original goldendict is also have this problem, 

Is it intended or not ,I can not conclude right now?

**Reply by michaelbeijer** [reply]
I made a quick video to explain what I mean: https://www.youtube.com/watch?v=6_O-xZY7vgc :-)

**Reply by xiaoyifang** [reply]
That is very clear now. Will implement this feature when time available.
I'll create a issue to record this.

**Reply by michaelbeijer** [reply]
great, thanks a lot!
